wolfssl-sys/build.rs

```diff
@@ -43,7 +43,10 @@ fn copy_wolfssl(dest: &Path) -> std::io::Result<PathBuf> {
 }
 
 const PATCH_DIR: &str = "patches";
-const PATCHES: &[&str] = &["disable-falcon-dilithium.patch"];
+const PATCHES: &[&str] = &[
+    "disable-falcon-dilithium.patch",
```

Comment:
Do we still need this falcon dilithium patch? We explicitly only enable kyber in the config

Reply:
Seems like we don't. After digging through `configure.ac`, both will only be defined if we enable dilthium and use liboqs. So I will remove this patch

Reply:
Actually we still need it lol cuz it turns out it won't store Private key if there's no Falcon or Dilithium... -,-

I think I will also file this bug to a separate ticket to them as well?

Reply:
Updated the bug ticket. I think we can merge this if this PR looks good since they are just going to put the update in the new PR anyway?

Reply:
I think at this stage its fine to patch it and maybe fix it up later if the future patch looks different

Reply:
They have just published a PR for it: https://github.com/wolfSSL/wolfssl/pull/8196
I have replaced the patch with their PR fix